src/Cns.ts

```diff
@@ -185,7 +185,9 @@ export default class Cns extends EthereumNamingService {
     values: string[],
   ): Record<string, string> {
     const records: Record<string, string> = {};
-    keys.forEach((key, index) => records[key] = values[index]);
+    keys.forEach((key, index) => {
+      if (!!values[index]) records[key] = values[index]
+    });
     return records;
```

Comment:
`Array#reduce` is not really working in this case, since it is outputting a single result when I want to check each element of keys arrays and add the specific ones to the records collection.

Reply:
```
return keys.reduce((r, k, i) => {...r, [k]: values[i]}, {})
```